### DISCUSSION TITLE
How do I get the Path segment to not show a tilde(~) for the home directory?

### DISCUSSION THREAD

**Discussion by jessey-git** [JanDeDobbeleer/oh-my-posh#3433] [General]
How do I get the Path segment to not show a tilde(~) for the home directory?

I often want to copy/paste the directory into other apps that do not understand `~` but I haven't been able to get the Path segment to cooperate.

I tried setting `home_icon` to an empty string as well as removing the property entirely but neither works. 

```
        {
          "background": "#306090",
          "foreground": "#ffffff",
          "properties": {
            "style": "full"
          },
          "style": "diamond",
          "template": " {{ .Path }} ",
          "trailing_diamond": "\ue0b0",
          "type": "path"
        },
```

**Comment by lewis-yeung** [comment]
@jessey-git See [docs](https://ohmyposh.dev/docs/segments/path#mapped-locations). You can add a line `"mapped_locations_enabled": false` under `"properties"` for this.

**Reply by jessey-git** [reply]
Oh indeed. I would have never guessed that. What a weird setting... especially considering that the "mapped_locations" property says "custom glyph/text for specific paths. Works regardless of the mapped_locations_enabled" :)

**Reply by lewis-yeung** [reply]
Setting `mapped_locations_enabled` to `false` only disables replacements for some **known locations**, and the Home directory to `~` is one of them. It doesn't affect the replacements defined in `mapped_locations`. :)
